docs/developer/hacking.rst

```diff
@@ -0,0 +1,51 @@
+#######
+Hacking
+#######
+
+.. note:: Before reading this file, please read the :doc:`kytos:contributing`
+    section that contains the main guidelines of the project.
+
+Development Environment Setup
+*****************************
+
+It is possible to install any of the *kytos* repos in develop mode, so that the
+installed files are kept in the original location, and only links are made to
+the system locations. This allows developpers to quickly test changes made in
+source code.
+
+To install a repo in develop mode, inside the repo source folder run:
+
+	.. code:: shell
+
+	    sudo python setup.py develop
```

Comment:
@diraol I have no idea if it has something to do with wheels... About the installation process, I believe it is not up to me to answer.